rush.json

```diff
@@ -363,7 +363,7 @@
       "versionPolicyName": "client"
     },
     {
-      "packageName": "@azure/quantum",
+      "packageName": "@azure/quantum-jobs",
       "projectFolder": "sdk/quantum/quantum",
```

Comment:
Shouldnt the folder name also change? i.e `sdk/quantum/quantum-jobs`

Reply:
Done